spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -169,8 +175,9 @@ def normal_refresh
       # oc delete limitRange my-limit-range-1
       # oc delete persistentVolumeClaim my-persistentvolumeclaim-1
       # oc delete template my-template-1
-      # oc delete build my-build-1
-      # oc delete buildconfig my-build-config-1
+      # oc delete build my-build-config-1
+      # oc delete buildconfig my-build-config
+      # oc delete rc/my-replicationcontroller-1
```

Comment:
Not necessarily related to this PR but we should use separate templates for the different scenarios instead of the above
(and delete them with  `oc process -f ... | oc delete -f -`)

That would be easier to automate cc @zakiva